99_master-kubelet-no-taint.yaml

```diff
@@ -31,13 +33,14 @@ spec:
                 --container-runtime=remote \
                 --container-runtime-endpoint=/var/run/crio/crio.sock \
                 --allow-privileged \
-                --node-labels=node-role.kubernetes.io/master \
+                --node-labels=node-role.kubernetes.io/master,node.openshift.io/os_id=${ID} \
```

Comment:
What is the `ID` here?

Reply:
```
$ cat /etc/os-release 
NAME="Red Hat Enterprise Linux CoreOS"
VERSION="410.8.20190710.0"
VERSION_ID="4.1"
PRETTY_NAME="Red Hat Enterprise Linux CoreOS 410.8.20190710.0 (Ootpa)"
ID="rhcos"
ID_LIKE="rhel fedora"
ANSI_COLOR="0;31"
HOME_URL="https://www.redhat.com/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"
REDHAT_BUGZILLA_PRODUCT="OpenShift Container Platform"
REDHAT_BUGZILLA_PRODUCT_VERSION="4.1"
REDHAT_SUPPORT_PRODUCT="OpenShift Container Platform"
REDHAT_SUPPORT_PRODUCT_VERSION="4.1"
OSTREE_VERSION=410.8.20190710.0
```

ID here `rhcos`

Reply:
Hmm Thanks,.. and this line sources that file: `EnvironmentFile=/etc/os-release`